EIPS/eip-7748.md

```diff
@@ -193,11 +193,11 @@ def state_convert(state: State, stride: int):
                 else:
                     # No more storage-slots. Move to the code migration starting
                     # at chunk-number zero.
```

Comment:
Given the CodePhase -> AccountDataPhase update, is this comment still accurate? Do we still start converting accounts at code chunk number 0

Reply:
Nope, was a left over. We move to "Account data" migration (i.e: all code, and account fields).